### ISSUE TITLE
DataGrid: Display issues when allow frozen columns to be resizable and column.colspan function exists

### ISSUE THREAD

**Issue by claudenbach** [infor-design/enterprise#5115]
Describe the bug
Display issues when allow frozen columns to be resizable and there exists column.colspan function.

1. Using the attached test-frozen-columns-resizable-with-colspan.html
2. Resize the width of the Product Type column 
3. Notice the blank column area that appears as its resized

Expected behavior
Allow frozen columns to be resizable even when a colspan exists.  The colspan represents the number of columns to span irrespective of whether frozen columns exist.

Version
- ids-enterprise:  v4.39.0-dev

Screenshots
![image](https://user-images.githubusercontent.com/39804038/115590178-4b79ce00-a296-11eb-804f-86e0000af16c.png)

**Comment by tmcconechy** [comment]
I'm guessing its going to be a time consuming/tricky one to fix.

**Comment by claudenbach** [comment]
[Git-5115.zip](https://github.com/infor-design/enterprise/files/6352604/Git-5115.zip)

**Comment by tmcconechy** [comment]
@tmcconechy  will have to prioritize this issue for landmark so adding to the sprint (i'm talking to myself 👍🏻 )

**Comment by tmcconechy** [comment]
@claudenbach i put in the example you gave me but i wasnt able to reproduce this. See my results:

https://drive.google.com/file/d/1ckpHJIW_iIRgiRsLqiUcB7vYABmB9ksx/view

**Comment by claudenbach** [comment]
I'm testing on v4.50.2 and still seeing the issue.  

![Git-5115](https://user-images.githubusercontent.com/39804038/116890509-51a36f00-abf3-11eb-83ce-6916e78c0c69.gif)

**Comment by tmcconechy** [comment]
Im testing on master which is 4.51-beta but it shouldn't be any different here... But i did try on windows and mac.
And i definitely am using the new example. But not seeing this.

**Comment by claudenbach** [comment]
Let me test with 4.51-beta and use test-frozen-columns-resizable-with-colspan.html.

**Comment by claudenbach** [comment]
Testing with v4.51.0-beta and using test-frozen-columns-resizable-with-colspan.html, I'm not able to duplicate either!

**Comment by tmcconechy** [comment]
Yeah weird - i wasn't expecting this would be magically fixed but great. Can't think of anything we put in since then that would be related.

Will just mark this as fixed in 4.51 and send to QA for QA please test http://localhost:4000/components/datagrid/test-frozen-columns-resizable-with-colspan

**Comment by claudenbach** [comment]
Well, when changing the overall browser window size, then I'm able to duplicate.

**Comment by tmcconechy** [comment]
I tried that on suspicion as well but it didn't come up for me. Maybe check the screen size in the elements tab in chrome dev tools and let me what dimension you see it? Any zoom enabled?

**Comment by claudenbach** [comment]
![Git-5115-More-Testing](https://user-images.githubusercontent.com/39804038/116895213-91b92080-abf8-11eb-8e9a-fba7788788d5.gif)

**Comment by claudenbach** [comment]
Is this what you were asking for?  Otherwise perhaps just give a call?

![image](https://user-images.githubusercontent.com/39804038/116895542-ef4d6d00-abf8-11eb-8272-b8dfee6d877d.png)

**Comment by tmcconechy** [comment]
Maybe we better after lunch... I tried your size (yes thats what i meant) and i am not seeing it. But in your video looks like your doing a second action that i cant make out like it works the first time then your doing something else?

**Comment by claudenbach** [comment]
Sounds good.  I'm unavailable today from 1:00pm to 2:00pm CST.

**Comment by tmcconechy** [comment]
https://main-enterprise.demo.design.infor.com/components/datagrid/test-frozen-columns-resizable-with-colspan.html now has your example...

**Comment by tmcconechy** [comment]
@claudenbach this fix is turning to be very tricky / risky and time consuming. Wondering if we can just restrict the resize in this scenario? I.E. If colspans and frozen columns disable resize.

I think we might need to redo the resize logic entirely which is planned for the new Web Component project.
